--- create_table.sh.orig	2015-09-30 16:28:09.655672576 +0000
+++ create_table.sh	2015-09-30 16:29:26.419672464 +0000
@@ -23,7 +23,7 @@
 # This can save a lot of storage space.
 DATA_BLOCK_ENCODING=${DATA_BLOCK_ENCODING-'DIFF'}
 DATA_BLOCK_ENCODING=`echo "$DATA_BLOCK_ENCODING" | tr a-z A-Z`
-TSDB_TTL=${TSDB_TTL-'FOREVER'}
+TSDB_TTL=${TSDB_TTL-'2592000'}

 case $COMPRESSION in
   (NONE|LZO|GZIP|SNAPPY)  :;;  # Known good.
