--- bin/hbase-daemon.sh.orig	(date 1597846164710)
+++ bin/hbase-daemon.sh	(date 1597846164710)
@@ -182,7 +182,14 @@
     hbase_rotate_log $logout
     hbase_rotate_log $loggc
     echo starting $command, logging to $logout
-    nohup $thiscmd --config "${HBASE_CONF_DIR}" internal_start $command $args < /dev/null > ${logout} 2>&1  &
+    echo "`date` Starting $command on `hostname`" >> $loglog
+    echo "`ulimit -a`" >> $loglog 2>&1
+    exec "$HBASE_HOME"/bin/hbase \
+        --config "${HBASE_CONF_DIR}" \
+        $command "$@" start
+    echo $! > $pid
+    wait
+    cleanZNode
     sleep 1; head "${logout}"
   ;;
