--- bin/start-hbase.sh.orig     2014-10-07 22:46:12.765559130 +0000
+++ bin/start-hbase.sh  2014-10-07 22:47:05.037560645 +0000
@@ -44,7 +44,7 @@ distMode=`$bin/hbase --config "$HBASE_CO
 
 if [ "$distMode" == 'false' ] 
 then
-  "$bin"/hbase-daemon.sh --config "${HBASE_CONF_DIR}" start master
+  exec "$bin"/hbase-daemon.sh --config "${HBASE_CONF_DIR}" start master
 else
   "$bin"/hbase-daemons.sh --config "${HBASE_CONF_DIR}" start zookeeper
   "$bin"/hbase-daemon.sh --config "${HBASE_CONF_DIR}" start master 
