--- bin/start-hbase.sh.orig	(date 1597840176574)
+++ bin/start-hbase.sh	(date 1597840176574)
@@ -52,7 +52,7 @@

 if [ "$distMode" == 'false' ]
 then
-  "$bin"/hbase-daemon.sh --config "${HBASE_CONF_DIR}" $commandToRun master $@
+  exec "$bin"/hbase-daemon.sh --config "${HBASE_CONF_DIR}" $commandToRun master $@
 else
   "$bin"/hbase-daemons.sh --config "${HBASE_CONF_DIR}" $commandToRun zookeeper
   "$bin"/hbase-daemon.sh --config "${HBASE_CONF_DIR}" $commandToRun master
